src/java.management/share/classes/javax/management/modelmbean/DescriptorSupport.java

```diff
@@ -1153,6 +1167,7 @@ private static Object parseQuotedFieldValue(String s)
                 "Cannot construct instance of " + className +
                 " with arg: <" + s + ">";
             throw new XMLParseException(e, msg);
+            // XMLParseException is deprecated for removal.
```

Comment:
Q: Non-deprecated method will be using the deprecated class `XMLParseException`. What is the plan for replacement? Should this be fixed now? The same question for the line 1160.

Reply:
Aside from the public deprecated API, the only places where this exception is used is in private static methods that support the implementation of the deprecated constructor, so these will go away when the constructor is eventually removed.

Reply:
If we wanted to be really cautious, we could hold off removal of the exception class.  But we should still deprecate it for removal now.
Code out there using specifically javax/management/modelmbean/XMLParseException would most likely be a mistake, it most likely meant some other XMLParseException class...  There really should not be any external usage...

Reply:
OK